pkg/cover/report.go

```diff
@@ -326,6 +326,15 @@ func (rg *ReportGenerator) lazySymbolize(files map[string]*file, progs []Prog) e
 	return nil
 }
 
+func (rg *ReportGenerator) GetSymbolsInfo() (symName []string, cuName []string, pcs [][]uint64) {
```

Comment:
Combine these 3 into a struct. 3 parallel arrays where indexes are assumed to match is too ad-hoc for a package interface.
```
type Symbol struct {
    Name string
    File string
    PCs []uint64
}

func (rg *ReportGenerator) GetSymbolsInfo() []Symbol {
```